config/repositories.json

```diff
@@ -53,5 +53,10 @@
     "GITHUB_REPO": "https://api.github.com/repos/pagopa/io-fixtures/git/ref/heads/",
     "BRANCH": "master",
     "ENV_COMMIT_SHA": "IO_FIXTURES_COMMIT_SHA"
+  },
+  {
+    "GITHUB_REPO": "https://api.github.com/repos/pagopa/io-functions-bpd/git/ref/heads/",
+    "BRANCH": "176068708-get-user-bpd",
```

Comment:
This could be changed to master when pagopa/io-functions-bpd#4 will be merged

Reply:
do it now ;)

Reply:
Fixed in 2046f6f